python/tests/recordio_test.py

```diff
@@ -77,6 +77,34 @@ def test_io_failures(self):
             recordio.Scanner("", index=idx)
         idx.close()
 
+    def test_utf8_values(self):
+        # filename can be in UTF-8
+        path = "/tmp/文件.recordio"
```

Comment:
As we are comparing the content read from the file with the content we wrote, we might need to call https://golang.org/pkg/io/ioutil/#TempFile to create the temporary file; otherwise, simultaneous writings by multiple test runs might conflict and ruin the content of the file.